plugin-vidarr/src/main/java/ca/on/oicr/gsi/shesmu/vidarr/RunStateMonitor.java

```diff
@@ -88,20 +93,35 @@ public RunStateMonitor(String workflowRunUrl, WorkflowRunStatusResponse status)
   }
 
   @Override
-  public boolean canReattempt() {
+  public AvailableCommands commands() {
+    // Failed during provision out may be retried
+    if (status.getEnginePhase() != null
+        && (status.getEnginePhase().equals("FAILED")
+            && status.getOperations() != null
+            && status.getOperations().stream()
+                .anyMatch(
+                    operation ->
+                        operation.getEnginePhase().equals("PROVISION_OUT")
+                            && operation.getStatus().equals("FAILED")))) {
+      return AvailableCommands.CAN_RETRY;
```

Comment:
While testing this PR, it looks like when a workflow run fails at provision out, both the "Reattempt Failed Workflow" and "Retry Provision Out" buttons are being displayed. Is this expected? It looks like here the buttons that should be displayed are being restricted to the relevant commands.

Reply:
Sure. I mean, is there a state when it can retry and there's a reason to reattempt? In #1272 I'm adding a ranking to the commands, so we can have retry is displayed first. If there's no good reason to reattempt, I'll change it.

Reply:
I can't think of a reason to reattempt (start the Vidarr workflow run from the beginning) when a Vidarr workflow run can be retried (failed at provision out).

Understood that the other PR adds ranking of retry and reattempt commands/operation/buttons, but it looks like here the available commands should be restricted to "CAN_RETRY", but I'm seeing action failed at provision file out with both retry and reattempt as available commands.

Another consideration is that failed actions can be worked on in bulk. It does not appear that the bulk "Reattempt Failed Workflow" and "Retry Provision Out" operation/buttons categorize the selected actions into can/should reattempt vs. can/should retry.

Reply:
Okay. Fixed.

Reply:
Thanks Andre, looks great!